ui/cypress/support/commands.ts

```diff
@@ -53,6 +60,35 @@ export const createCell = (
   })
 }
 
+export const createView = (
+  dbID: string,
+  cellID: string
+): Cypress.Chainable<Cypress.Response> => {
+  return cy.fixture('view').then(view => {
+    return cy.request({
+      method: 'PATCH',
+      url: `/api/v2/dashboards/${dbID}/cells/${cellID}/view`,
+      body: view,
+    })
+  })
+}
+
+export const createDashWithCell = (
+  orgID: string
+): Cypress.Chainable<Cypress.Response> =>
+  createDashboard(orgID).then(({body: dashboard}) => createCell(dashboard.id))
+
+export const createDashWithView = (
```

Comment:
what do you think about changing this function name to indicate that it's also creating a variable? along the lines of `createDashWithViewAndVariable` ?